lms/djangoapps/discussion/templates/discussion/edx_ace/responsenotification/push/body.txt

```diff
@@ -0,0 +1,2 @@
+{% load i18n %}
+{% blocktrans trimmed %}{{ comment_username }} replied to {{ thread_title }}{% endblocktrans %}
```

Comment:
So, we're not including any excerpt? Depending on how much mobile notifications limit we have in mobile phones.

I'd recommed to add the comment's first 200 characters or so:

```suggestion
{% blocktrans trimmed %}{{ comment_username }} replied to {{ thread_title }}: {{ comment_body_excerpt }}{% endblocktrans %}
```

`comment_body_excerpt` would be either the whole comment or part of it ending with `...`.

Reply:
In general, this was not in the text of the push notifications when they were approved, but if you insist, I added it.